xbmc/pvr/PVRManager.h

```diff
@@ -706,6 +706,8 @@ namespace PVR
     ManagerState                    m_managerState;
     std::unique_ptr<CStopWatch>     m_parentalTimer;
 
+    CCriticalSection                m_initMutex; // mutex for protecting pvr manager's start/restart sequence */
```

Comment:
`m_startStopMutex` would be a more appropriate name as it serializes only these two function.